website/public/js/services/groupServices.js

```diff
@@ -94,10 +94,10 @@ angular.module('habitrpg')
       });
     };
 
-    Group.startQuest = function(gid) {
+    Group.startQuest = function(gid, key) {
       return $http({
         method: "POST",
-        url: groupApiURLPrefix + '/' + gid + '/questAccept',
+        url: groupApiURLPrefix + '/' + gid + '/quests/invite/' + key,
```

Comment:
hmm not sure this is correct. If it's only used to invite to a quest it should be called Group.inviteToQuest